### DISCUSSION TITLE
Frigate+ new label: Umbrella?

### DISCUSSION THREAD

**Discussion by nuansed** [blakeblackshear/frigate#3521] [Ideas]
In Frigate I currently also detect Umbrellas and treat those detections as equivalent to people in any automations. 
![snapshot-2](https://user-images.githubusercontent.com/12543435/180678300-49d86442-5dc3-4e82-9aa8-d9cb80adab8e.jpg)
![snapshot-3](https://user-images.githubusercontent.com/12543435/180678593-41717bb1-3872-4df9-9fda-8345af72bdaa.jpg)

In Frigate+, should I label people where most of their body is covered by an umbrella as "person"? Or would it be better to label them with a new label "umbrella"?

**Comment by NickM-27** [comment]
I think @blakeblackshear will have a better answer, but my intuition is that the the top image would be labeled as nothing and the bottom image could be a "person" with the `difficult` label checked.

**Comment by blakeblackshear** [comment]
Label persons that are partially visible. Don't label umbrellas as a person. I had never thought of this use case. I will consider adding umbrella in the next round of additions to Frigate+.

**Comment by nuansed** [comment]
Ok, will do. When exactly should I tick "difficult"? What does it do in the backend?

**Reply by NickM-27** [reply]
Quoting https://github.com/blakeblackshear/frigate/discussions/1043#discussioncomment-2313486 : 

> Difficult is a bit subjective, but I would say no on that one. It will be used when evaluating performance and possibly with training. I would consider a person barely visible through a tree or bush to be difficult. Basically anything you wouldn't expect a machine to be able to recognize.
